Integrations/integration-SandBlastAppliance.yml

```diff
@@ -0,0 +1,450 @@
+description: Query, upload and download data using Check Point Sandblast
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Version
+  name: version
+  defaultvalue: v1
+  type: 0
+  required: true
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |-
+    var server = params.server;
+    var insecure = params.insecure;
+    var proxy = params.proxy;
+    var version = params.version;
+    var requestTemplate = JSON.stringify({request:[{features:['te']}]});
+    var base = server.replace(/[\/]+$/, '') +':18194/tecloud/gw/' + version + '/file/';
+
+    var urlDict = {
+        'sb-query': 'query',
+        'sb-upload': 'upload',
+        'sb-download': 'download',
+        'sandblast-query': 'query',
+        'sandblast-upload': 'upload',
+        'sandblast-download': 'download'
+    };
+
+    var sendRequest = function(url, body, queryName) {
+        log('URL IS: '+base + url)
+        log('BODY IS: '+JSON.stringify(body))
+        var res = http(
+                base + url,
+                {
+                    Method: 'POST',
+                    Body: body
+                },
+                insecure,
+                proxy
+            );
+        log('Status code is: '+res.StatusCode )
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Failed to ' + queryName + ' , request status code: ' + res.StatusCode + ' and Body: ' + res.Body + '.';
+        }
+        return JSON.parse(res.Body);
+    };
+
+    var teArgs = {'images': 0, 'reports': 0, 'benign_reports': 1};
+
+    var createRequest = function(args) {
+        if (!args.features) {
+            args.features = 'all';
+        }
+        if (args.features !== 'extraction' && args.features !== 'av') {
+            var feature = {};
+                var keys = Object.keys(teArgs);
+                for (var i = 0; i < keys.length; i++) {
+                    if (args[keys[i]]) {
+                        if (teArgs[keys[i]] === 0) {
+                            feature[keys[i]] = JSON.parse(args[keys[i]]);
+                        } else if (teArgs[keys[i]] === 1) {
+                            feature[keys[i]] = args[keys[i]] === 'true';
+                        }
+                        delete args[keys[i]];
+                    }
+                }
+                if (Object.keys(feature).length) {
+                    args['te'] = feature;
+                }
+        }
+        if (args.features === 'all') {
+            args.features = ['te', 'av', 'extraction'];
```

Comment:
Don't we need only `te` feature only here?

Reply:
No, we need them all.